### DISCUSSION TITLE
Welcome to jbang Discussions!

### DISCUSSION THREAD

**Discussion by maxandersen** [jbangdev/jbang#257] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other jbang users. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other jbang users and developers.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do or want to do with jbang!

**Comment by OLibutzki** [comment]
Hi,

I'm Oliver.

In my current project I try to convince my colleagues that jbang is a great tool to enhance our Jenkins deployment pipelines. Because of #245 the start was a little bit bumpy. 😉

The first thing I built is a script which preconfigures a Keycloak instance for our application.

I feel much more familiar in doing this stuff by using a statically typed language like Java than using bash or Groovy.

**Reply by maxandersen** [reply]
Welcome! And sorry for the space for issue. I swore we wouldn’t have that issue after we fixed it the first time. Hopefully 0.39 works for you :)

**Reply by maxandersen** [reply]
Btw. How do you use it from Jenkins ? Been considering if it made sense to make a jbang plugin for Jenkins to make it easier to get started.

**Reply by OLibutzki** [reply]
We use Docker build agents with a custom image. Into this image we install the tools we need like JDK, maven and jbang.

**Reply by maxandersen** [reply]
Ok. Then you won’t have much use of a Jenkins plugin

**Comment by gastaldi** [comment]
Hi,

I'm George.

I am using JBang with GitHub Actions to enhance our CI pipelines. I :heart: JBang.

For me, having a [JBang Catalog](https://github.com/jbangdev/jbang#implicit-alias-catalogs) is a killer feature, since this allows me keep a JBang script updated in a single place.

Also Java > any other language (Change my mind) :laughing:

**Reply by quintesse** [reply]
I've tried to change your mind, but you're too stubborn 😄

**Comment by quintesse** [comment]
Hi,

I'm Tako and I knewJbang was awesome from the moment I saw it. I always wanted to be able to use Java for those little scripting problems. The ones where setting up a Java project is way too much work. Jbang just makes that effortless.

In general I've been helping Max wherever I can to make Jbang even better.
